udata/core/discussions/csv.py

```diff
@@ -0,0 +1,26 @@
+from udata.frontend import csv
+
+from .models import Discussion
+
+
+@csv.adapter(Discussion)
+class DiscussionCsvAdapter(csv.Adapter):
+    fields = (
+        "id",
+        "user",
+        "subject",
+        ("subject_class", "subject.class"),
+        ("subject_id", "subject.id"),
+        "title",
+        ("size", lambda o: len(o.discussion)),
+        (
+            "participants",
+            lambda o: ",".join(
+                getattr(msg.posted_by, "class") + "_" + msg.posted_by.id for msg in o.discussion

```

Comment:
Why using a `getattr`? For now posted_by is only a user, I'm unsure about the way it would evolve if it can be an Organization, but I think you should not anticipate this here yet.

Maybe add a comment on https://github.com/datagouv/data.gouv.fr/issues/1410 to mention to update the adapter at that time?

Reply:
When using the usual form `msg.posted_by.class` my editor displayed an error because `class` was recognized as a reserved keyword (but I am not sure it would have failed actually). I'll only get the ids, as indeed we only have users for now